packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -661,39 +765,172 @@ abstract class TableBase extends Resource implements ITable {
    *
-   * @default sum over a minute
+   * By default, the metric will be calculated as a sum over a period of 5 minutes.
+   * You can customize this by using the `statistic` and `period` properties.
    */
   public metricConditionalCheckFailedRequests(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
     return this.metric('ConditionalCheckFailedRequests', { statistic: 'sum', ...props });
   }
 
   /**
-   * Metric for the successful request latency this table
+   * Metric for the successful request latency this table.
    *
-   * @default avg over a minute
+   * @deprecated use `metricSuccessfulRequestLatencyForOperations`
    */
   public metricSuccessfulRequestLatency(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+
+    if (!props?.dimensions?.Operation || !props?.dimensions?.TableName) {
+      // Operation must be passed because its an operational metric.
+      // TableName must be passed because the dimensions here will override the default ones which contain the table name.
+      throw new Error("Both 'Operation' and 'TableName' dimensions must be passed for the 'SuccessfulRequestLatency' metric.");
+    }
+
     return this.metric('SuccessfulRequestLatency', { statistic: 'avg', ...props });
   }
 
+  /**
+   * Metric for the system errors this table.
+   *
+   * This will sum errors across all possible operations.
+   * Note that by default, each individual metric will be calculated as a sum over a period of 5 minutes.
+   * You can customize this by using the `statistic` and `period` properties.
+   */
+  public metricSystemErrorsForOperations(props?: SystemErrorsForOperationsMetricOptions): cloudwatch.IMetric {
+
+    if (props?.dimensions?.Operation) {
+      throw new Error("The Operation dimension is not supported. Use the 'operations' property.");
+    }
+
+    const operations = props?.operations ?? Object.values(Operation);
+
+    const values = this.createMetricsForOperations('SystemErrors', operations, { statistic: 'sum', ...props });
+
+    const sum = new cloudwatch.MathExpression({
+      expression: `${Object.keys(values).join(' + ')}`,
+      usingMetrics: { ...values },
+      color: props?.color,
+      label: props?.label ?? 'Sum over all Operations',
+      period: props?.period,
+    });
+
+    return sum;
+  }
+
+  /**
+   * Metric for the successful request latency for specific operations of this table.
+   *
+   * This will perform a weighted average across all possible operations.
+   * Note that by default, each individual metric will be calculated as an average over a period of 5 minutes.
+   * You can customize this by using the `statistic` and `period` properties.
+   */
+  public metricSuccessfulRequestLatencyForOperations(props: SuccessfulRequestLatencyForOperationsMetricOptions): cloudwatch.IMetric {
+
+    if (props?.dimensions?.Operation) {
+      throw new Error("The Operation dimension is not supported. Use the 'operations' property.");
```

Comment:
Any suggestion for additional context here to help users figure out exactly which call is causing this? Just add the method name to the message?

Reply:
Sure. 
```suggestion
      throw new Error("the 'Operation' dimension is not supported for 'metricSuccessfulRequestLatencyForOperations'; use the 'operations' property.");
```

Reply:
Meh. The stack trace will tell them.

Reply:
Stack traces only show up in verbose...not the best experience. I added the method name.